server/app/routes/kits.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+var router = require('express').Router();
+var Kit = require('../../../db/models/kit');
```

Comment:
I think that the path on line 3 for requiring models kit is going to give an error. I think it should be `require('../db/models')`